### ISSUE TITLE
Raise max listeners for electron

### ISSUE THREAD

**Issue by cdrage** [containers/podman-desktop#7764]
### Is your enhancement related to a problem? Please describe

![Screenshot 2024-06-20 at 4 37 54 PM](https://github.com/containers/podman-desktop/assets/6422176/8b714896-6ace-4d4c-aebc-d1459921207a)

Getting this each time run Podman Desktop now.

Unsure if this is causing any problems however.. seems like everything is working okay.

### Describe the solution you'd like

Should we raise the max listeners?

**Comment by cdrage** [comment]
cc @benoitf

**Comment by benoitf** [comment]
yes it's a warning but I'll check, probably now with more extensions we need to configure some with more listeners
